py/riscv/PcConfig.py

```diff
@@ -16,7 +16,7 @@
 
 
 def get_reset_pc():
-    return 0x50000000
+    return 0x80000000
```

Comment:
I think you are right. I'll see if there is a clean way to do that. I thought of copying, updating, and using it but since riscv-cva6.config does not access it directly, I need to find some alternative. Meanwhile I'll make the other changes suggested. Thanks for responding promptly,